packages/fxa-auth-server/README.md

```diff
@@ -156,6 +156,42 @@ Rate-limiting and blocking is handled by fxa-customs-server. By default, these p
 
 The customs-server can be enabled for local testing by changing the dev config to `"customsUrl":"http://localhost:7000"`.
 
+### Recovery Phone Config
+
+Recovery phone uses twilio to send SMS. Twilio has two different modes it can operate in based on the credentials provided.
+
+The 'test' mode can send SMS to magic test numbers at no cost. This is what we use for unit/integration testing.
+
+The 'default' mode can send SMS to real numbers, but will incur const, and cannot send messages to magic test numbers. This
```

Comment:
Add explanation for `default` vs `apiKeys` modes, as well as short note on how to obtain these credentials?